cli/src/main/kotlin/io/redgreen/tumbleweed/cli/dev/view/ViewCommand.kt

```diff
@@ -29,6 +31,10 @@ class ViewCommand : Runnable {
   var port: Int = DEFAULT_PORT
 
   override fun run() {
+    if (!jsonFile.toPath().exists()) {
+      throw IllegalArgumentException("JSON file not found at given path: $jsonFile, please provide a valid file path")
+    }
```

Comment:
Can you help me understand "we cannot read the file as JSON string"?

This is what I had in mind,
Since we are validating inputs to the command, we do the following checks in sequence.

1. The file actually exists and it is not a directory.
2. `jsonFile.readText()` to read the contents as text.
3. Try validating this JSON text through a companion object function like `BilevelEdgeBundlingGraph.isValidJson(json: String): Boolean`.

If any of the checks fails, we print to the console using `println()`, instead of the logger. Also, FYI the logging libraries are available for all modules.

Am I missing something here?

Reply:
Nope. That's it, it can be an addition check to file check. Should I go ahead with that implementation?

Reply:
Yes please 😊